common/utils/src/main/resources/error/error-states.json

```diff
@@ -2933,6 +2933,12 @@
         "standard": "Y",
         "usedBy": ["SQL/Foundation", "PostgreSQL", "Redshift", "Oracle", "SQL Server"]
     },
+    "42001": {
+        "description": "Invalid encoder error",
+        "origin": "SQL/Foundation",
+        "standard": "Y",
+        "usedBy": ["SQL/Foundation"]
```

Comment:
Could you check is this SQL state used by other DBMS?

Reply:
I took a look at different DBMS sqlstates and to my knowledge it seems that this one is not used by any. Changed doc accordingly.